.travis-mirage.sh

```diff
@@ -0,0 +1,57 @@
+## mirage setup
+
+set -ex
+
+## fetch+execute the OCaml/opam setup script
+wget https://raw.githubusercontent.com/mor1/ocaml-travisci-skeleton/master/.travis-ocaml.sh
+sh .travis-ocaml.sh
+
+## install mirage
+export OPAMYES=1
+eval $(opam config env)
+opam install mirage
+
+## copy travis `config.ml` in place
+cp .travis-config.ml src/config.ml
```

Comment:
i guess i'm avoiding using it directly mostly because i'm following existing pattern :) for which the underlying reason seems to be that net config is controlled by `DHCP=[false|true]` which isn't enough (the `false` case then needs to distinguish between a static ip and using default ip). i'll fix that in these three repos so that using the `DEPLOY` flag takes care of it (since if that flag is present, it indicates a static ip). so i can remove the `cp` i think.